packages/cli/src/commands/start.ts

```diff
@@ -239,6 +239,14 @@ export class Start extends Command {
 		const { flags } = this.parse(Start);
 
 		try {
+			// Load all node and credential types
```

Comment:
We had to move the loading of nodes and credentials up to make sure they are available in the migrations. @netroy Do you see any issues with this?